x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/use_create_analytics_form.ts

```diff
@@ -285,7 +285,7 @@ export const useCreateAnalyticsForm = (): CreateAnalyticsFormProps => {
     resetForm();
     const config = extractCloningConfig(cloneJob);
     if (isAdvancedConfig(config)) {
-      setJobConfig(config);
+      setFormState(getFormStateFromJobConfig(config));
```

Comment:
If I clone the job created using the config in https://github.com/elastic/kibana/pull/79048#discussion_r498193432, the `query` part does not appear to get copied over in the clone. The advanced wizard is opening with the other parts of the config copied over, but the query seems to have been reset to the default:

![image](https://user-images.githubusercontent.com/7405507/94913289-ea78dd00-04a0-11eb-8e52-41fd1380bb25.png)

Reply:
Good find - @peteharverson - looks like the job config query wasn't properly getting saved in the form state. Fixed in 1c3bbadaab4247ce85eca8f240440ee3c895b671